core/dbt/contracts/graph/semantic_manifest.py

```diff
@@ -58,6 +60,20 @@ def validate(self) -> bool:
         semantic_manifest = self._get_pydantic_semantic_manifest()
         validator = SemanticManifestValidator[PydanticSemanticManifest]()
         validation_results = validator.validate_semantic_manifest(semantic_manifest)
+        new_time_spines = semantic_manifest.project_configuration.time_spines
```

Comment:
can we just name this as `time_spines`, and rename `old_time_spines` to `legacy_time_spines`? 'new' will have little semantic meaning outside the context of this PR. we also do a bunch of state comparison operations throughout the codebase so a reader may mistake it for something like that at first.